src/ensureLocationMiddleware.js

```diff
@@ -0,0 +1,65 @@
+import ActionTypes from './ActionTypes';
+
+function ensureLocation(location) {
+  if (typeof location === 'object') {
```

Comment:
maybe `!== 'string'` instead? more options for exotic descriptors? ore of a "why not" than me having any real use case